### PR TITLE
Issue 4 add to database - Aaron Butnik

### PR DESCRIPTION
**Pull request by aaron-j-butnik**
issue-4-moblie-view
rick-morty-issue-4-ajax-desktop-view

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -126,3 +129,87 @@ function handleLiClick(event) {
...
+
+$btnAdd.addEventListener('click', handleAddToDatabase);
+
+function handleAddToDatabase(event) {
+
+  var databaseCharacter = event.target.closest('[data-character-id]');
+
+  for (var i = 0; i < data.entries.length; i++) {
+    if (Number(databaseCharacter.getAttribute('data-character-id')) === data.entries[i].charID) {
+      data.database = data.entries[i];
```
Comment (codingcodymiller) [inline js/main.js]: Does this only ever store one entry? What if the user tries to save multiple entries, are they just forgotten?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by aaron-j-butnik** [comment]
image
image

File: js/main.js [hunk 2]
```diff
@@ -142,10 +142,14 @@ function handleAddToDatabase(event) {
...
     if (Number(databaseCharacter.getAttribute('data-character-id')) === data.entries[i].charID) {
-      data.database = data.entries[i];
+      data.database.push(data.entries[i]);
     }
   }
-  $divDatabaseCharImg.appendChild(getPersonalDatabaseImgLi(data.database));
+
+  for (var j = 0; j < data.database.length; j++) {
+    var entryFromDatabase = data.database[j];
+  }
```
Comment (codingcodymiller) [inline js/main.js]: Are you doing this just to get the last item in the array? You should do `data.database[data.database.length - 1]` instead

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by aaron-j-butnik** [comment]
image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**